src/main/java/de/ellpeck/actuallyadditions/mod/config/values/ConfigIntValues.java

```diff
@@ -79,6 +81,22 @@
         this.desc = desc;
     }
 
+    public void initializeValue(Configuration config){
+        Property property = config.get(this.category, this.name, this.defaultValue, this.desc, this.min, this.max);
+        int value = property.getInt();
+        if (value < this.min)
+        {
+            value = this.min;
+            property.set(this.min);
+        }
+        if (value > this.max)
+        {
+            value = this.max;
+            property.set(this.max);
+        }
+        this.currentValue = value;
+    }
+
```

Comment:
So this whole method here is useless, and should just be config.getInt and use the min/max values of that method.

Reply:
```Configuration``` class (by Forge) uses minimum and maximum values just for displaying purpose. When it gets values - it ignores ones. Check methods ```getInt()```, ```getDouble()``` and so on. These methods *completely* ignore minimum and maximum values. That's because I added these checks.

Reply:
Those methods do not ignore the bounds given to them.  `getInt` and `getFloat` have checks in place.  And well, apparently there is no `getDouble`

Reply:
Please, show me, where is this internal check. (I checked whole ```Configuration``` and ```Property``` classes and didn't found; I found it only in GUI configuration classes)